lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/KBVSessionItem.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.di.ipv.cri.kbv.api.domain;
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+public class KBVSessionItem {
```

Comment:
Since authorization code and tokens etc are held in the sessionItem class, what is this class now for?

If it is for holding KBV specific data, I suggest it is renamed